Layout-with-Position/css-positioning_layout-with-position/css/styles.css

```diff
@@ -68,14 +86,20 @@ body {
   padding: 6px 12px;
   border-radius: 50%;
   box-shadow: rgba(0, 0, 0, 0.24) 0 3px 8px;
+  position: absolute;
+  top : 30px;
+
+  position: sticky;
```

Comment:
You have to decide: position: absolute or position: sticky. (the first one is correct ;) )